### PR TITLE
Improved exception logging and cleanup in runner initializers

### PR DESCRIPTION
**Pull request by srfarley**
We were letting exceptions propagate to the thread, and the stack traces were being dumped to stderr, interleaved with log messages.

I also moved the "helper" classes inline to the classes they were helping since they aren't very complex and are specific to the needs of the enclosing classes.

- [x] @sfeng88 @rygim @jharwig 
- [x] @joeferner @mwizeman 
- [x] @EvanOxfeld @joeybrk372 @dsingley

### CODE REVIEW

**Comment by srfarley** [comment]
@mwizeman @rygim @sfeng88 Are you OK with this merging today?

**Comment by sfeng88** [comment]
Haven't had a chance to review this yet. Probably won't get to it till tomorrow. Do you need it today?

**Comment by srfarley** [comment]
@sfeng88 I can use this branch locally for another day.

File: web/web-base/src/main/java/org/visallo/web/initializers/GraphPropertyWorkerRunnerInitializer.java [hunk 1]
```diff
@@ -34,13 +37,37 @@ public void initialize(ServletContext context) {
...
+                    } catch (Exception ex) {
+                        LOGGER.error("Failed running GraphPropertyRunner", ex);
+                    }
+                }
+
+                @Override
+                public void stop() {
+                    try {
+                        if (graphPropertyRunner != null) {
+                            graphPropertyRunner.stop();
```
Comment (sfeng88) [inline web/web-base/src/main/java/org/visallo/web/initializers/GraphPropertyWorkerRunnerInitializer.java]: Can we add a log statement here saying that we are stopping the graph property runner?

File: web/web-base/src/main/java/org/visallo/web/initializers/LongRunningProcessRunnerInitializer.java [hunk 2]
```diff
@@ -30,9 +33,33 @@ public void initialize(ServletContext context) {
...
+                    } catch (Exception ex) {
+                        LOGGER.error("Failed running LongRunningProcessRunner", ex);
+                    }
+                }
+
+                @Override
+                public void stop() {
+                    try {
+                        if (longRunningProcessRunner != null){
+                            longRunningProcessRunner.stop();
```
Comment (sfeng88) [inline web/web-base/src/main/java/org/visallo/web/initializers/LongRunningProcessRunnerInitializer.java]: Same as above but for long running process

**Comment by sfeng88** [comment]
Other than those two comments it looks good to me.

**Comment by srfarley** [comment]
@sfeng88 I added the logging.

**Pull request merged**